types/evidence.go

```diff
@@ -181,13 +181,15 @@ func init() {
 type DuplicateVoteEvidence struct {
 	VoteA *Vote `json:"vote_a"`
 	VoteB *Vote `json:"vote_b"`
+
+	Timestamp time.Time `json:"timestamp"`
```

Comment:
should we add a comment here indicating that this is the time of the header where this evidence was committed?

Reply:
Yeah sure we can add that although I think committed is a tricky word because it also could mean when evidence was committed on the chain or when the attack was committed

Reply:
feel free to change the wording